### ISSUE TITLE
Graphics issue related to issue 12

### ISSUE THREAD

**Issue by sluggishsloth** [NicolasGuilloux/blade-shadow-beta#79]
**Describe the bug**
Not sure how to reference other issues, but since updating to Ubuntu 20.04 focal, my lower screen is not rendering, and my inputs are not being recognized

**System**
 - OS: Ubuntu
 - OS Version: 20.04
 - Environment: Gnome
 - Application: Shadow Official and Shadow Beta AppImage
 - Application Version: 3.7.1

**Configuration**
 - The user is in input group? Yes
 - The environment uses Xorg? Yes
 - VA API detects H264 ? And H265? Yes

**To Reproduce**
1. Launch Shadow AppImage on Ubuntu 20.04
2. Screen renders improperly

VaInfo:
shai@shai-Revolve-810-G1:~$ vainfo
libva info: VA-API version 1.7.0
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_7
libva error: /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so init failed
libva info: va_openDriver() returns 1
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/i965_drv_video.so
libva info: Found init function __vaDriverInit_1_6
libva info: va_openDriver() returns 0
vainfo: VA-API version: 1.7 (libva 2.6.0)
vainfo: Driver version: Intel i965 driver for Intel(R) Ivybridge Mobile - 2.4.0
vainfo: Supported profile and entrypoints
      VAProfileMPEG2Simple            :	VAEntrypointVLD
      VAProfileMPEG2Simple            :	VAEntrypointEncSlice
      VAProfileMPEG2Main              :	VAEntrypointVLD
      VAProfileMPEG2Main              :	VAEntrypointEncSlice
      VAProfileH264ConstrainedBaseline:	VAEntrypointVLD
      VAProfileH264ConstrainedBaseline:	VAEntrypointEncSlice
      VAProfileH264Main               :	VAEntrypointVLD
      VAProfileH264Main               :	VAEntrypointEncSlice
      VAProfileH264High               :	VAEntrypointVLD
      VAProfileH264High               :	VAEntrypointEncSlice
      VAProfileH264StereoHigh         :	VAEntrypointVLD
      VAProfileVC1Simple              :	VAEntrypointVLD
      VAProfileVC1Main                :	VAEntrypointVLD
      VAProfileVC1Advanced            :	VAEntrypointVLD
      VAProfileNone                   :	VAEntrypointVideoProc
      VAProfileJPEGBaseline           :	VAEntrypointVLD

**Comment by aarnoud** [comment]
Sadly with those iGPU (Intel HD3000-4600) you need either Ubuntu 18.04 or Shadowcker (https://gitlab.com/aar642/shadowcker)

Duplicate of #12

**Comment by sluggishsloth** [comment]
Thanks! I deployed Shadowcker, which definitely fixes the graphics. I'm now experiencing a separate issue, which I have logged on your Gitlab page.

I know that Shadow's official site  lists 18.04 as a requirement. 
`sudo curl https://raw.githubusercontent.com/NicolasGuilloux/blade-shadow-beta/master/resources/drirc -o /etc/drirc`
did not work for me.

Can we append [https://nicolasguilloux.github.io/blade-shadow-beta/issues#the-drirc-fix](https://nicolasguilloux.github.io/blade-shadow-beta/issues#the-drirc-fix) to emphasize the 18.04 requirement, and add the [Shadowcker](https://gitlab.com/aar642/shadowcker/) workaround?
I definitely want to encourage more Linux users to adopt Shadowcker instead. Plus, with Shadowcker I can use other distros :+1: 

Thanks again!
